tests/foreman/ui/test_contentview.py

```diff
@@ -17,6 +17,71 @@
 
 from robottelo.constants import FAKE_FILE_NEW_NAME, REPOS, DataFile
 
+VERSION = 'Version 1.0'
+
+
+@pytest.mark.tier2
+@pytest.mark.upgrade
+def test_positive_end_to_end(session, module_target_sat, module_org, module_lce, target_sat):
+    """Create content view with yum repo, publish it and promote it to Library
+
+    :id: 0970db8d-e9e3-4317-8703-153a6d9b2875
+
+    :steps:
+        1. Create Product/repo and Sync it
+        2. Create CV and add created repo in step1
+        3. Publish and promote it to new LCE
+
+    :expectedresults: content view is created, updated with repo publish and
+        promoted to  the LCE
+
+    :CaseImportance: High
+    """
+    repo_name = gen_string('alpha')
+    cv_name = gen_string('alpha')
+    # Creates a CV along with product and sync'ed repository
+    target_sat.api_factory.create_sync_custom_repo(module_org.id, repo_name=repo_name)
+    with module_target_sat.ui_session() as session:
+        session.organization.select(org_name=module_org.name)
+        session.contentview_new.create({'name': cv_name})
+        session.contentview_new.add_content(cv_name, repo_name)
+        result = session.contentview_new.publish(cv_name, promote=True, lce=module_lce.name)
+        assert result[0]['Version'] == VERSION
+        assert module_lce.name in result[0]['Environments']
+
+
+@pytest.mark.tier2
+def test_positive_ccv_e2e(session, module_target_sat, module_org, module_lce, target_sat):
```

Comment:
```suggestion
def test_positive_ccv_e2e(session, module_target_sat, module_org, module_lce):
```
Same as above. Or do I actually overlook something? :thinking: